UselessMine.cpp

```diff
@@ -476,24 +492,28 @@ bool UselessMine::SlashCommand(int playerID, bz_ApiString command, bz_ApiString
     }
     else if (command == "reload" && bz_hasPerm(playerID, "setAll"))
     {
-        if (params->get(0) == "deathmessages")
-        {
-            reloadDeathMessages();
-            bz_sendTextMessage(BZ_SERVER, playerID, "Death messages reloaded");
-            return true;
-        }
-        if (params->get(1) == "defusalmessages")
-        {
-            reloadDeathMessages();
-            bz_sendTextMessage(BZ_SERVER, playerID, "Defusal messages reloaded");
-            return true;
-        }
-        
         if (params->size() == 0)
         {
             reloadDeathMessages();
             reloadDefusalMessages();
         }
+
+        else
+        {
+            if (params->get(0) == "deathmessages")
+            {
+                reloadDeathMessages();
+                bz_sendTextMessage(BZ_SERVER, playerID, "Death messages reloaded");
+                return true;
+            }
+
+            if (params->get(1) == "defusalmessages")
```

Comment:
There are only two possibilities here: `/reload deathmessages` and `/reload defusalmessages`. An elseif and only check for index 0. By getting index 1, you'd be looking for something like `/reload somethinghere defusalmessages`